### ISSUE TITLE
Proper way to run inferrence using models?

### ISSUE THREAD

**Issue by JakobKHAndersen** [nabsabraham/focal-tversky-unet#9]
Hello

Firtst of all thanks for creating this repository. I have a question regarding the use of the of the multi-scale models for segmentation of test set images. I have succesfully trained the model and want to use on my test data. If i understand the model correctly, the output is a list containing 4 arrays of different resolutions (original input size and 3 down-scaled versions, yes?). Do i only use the sigmoid/softmax probability map of the  original input sized output, or do i use all 4 and perform some sort of resizing with interpolation and average across in order to get to full benefit of the models? 

BR.

**Comment by nabsabraham** [comment]
Hello! I just use the last preds which are similar size as the input [[see here]](https://github.com/nabsabraham/focal-tversky-unet/blob/347d39117c24540400dfe80d106d2fb06d2b99e1/bus_train.py#L119) but you could upsample the smaller scale predictions and do some sort of weighted average or fusion to get a better result. The line concerning `preds_up` after is just to resize the prediction to the original input image (because they were really large).
